### ISSUE TITLE
Bot don't care the most important in game -> base

### ISSUE THREAD

**Issue by tienhoang1094** [adamqqqplay/dota2ai#86]
When all things destroyed and only ancient base left, creep or hero attacking it, bots dont care and still farm or run around. this bug apear with almost all of AI version. Could you fix that ?

**Comment by adamqqqplay** [comment]
Thanks for your feedback. It looks like some issues in tower defending mode. We will try to find the cause, and fix it.

**Comment by megastruktur** [comment]
@adamqqqplay you could try to replicate the same logic like when they defend T1  :D

**Comment by mange-ess** [comment]
Yep, had a pretty silly situation a moment ago playing on unfair level. Although all five heroes of the AI were alive and around their home base, they couldn't care less about two of my creeps hacking away at their ancient for about a minute until it was destroyed. 🤣
